app/services/katello/pulp3/content_view_version/importable_repositories.rb

```diff
@@ -101,14 +110,20 @@ def create_repo_params(metadata_repo, product)
 
           params = { name: find_unique_name(metadata_repo, product) }
           params[:gpg_key_id] = gpg_key_id(metadata_repo)
-
           keys.each do |key|
             params[key] = metadata_repo.send(key)
           end
-
+          url = fetch_feed_url(metadata_repo)
+          params[:url] = url if url
           params
         end
 
+        def fetch_feed_url(metadata_repo)
+          return unless @syncable_format
+          path = @path.split('/').join('/')
```

Comment:
Speaking offline, it seems the intent is to trim the trailing slash. chomp('/') would make more sense to do that.